azkaban-common/src/main/java/azkaban/executor/mail/DefaultMailCreator.java

```diff
@@ -130,12 +130,12 @@ public boolean createErrorEmail(ExecutableFlow flow, EmailMessage message,
     if (emailList != null && !emailList.isEmpty()) {
       message.addAllToAddress(emailList);
       message.setMimeType("text/html");
-      message.setSubject("Flow '" + flow.getFlowId() + "' has failed on "
-          + azkabanName);
+      message.setSubject("Flow '" + flow.getFlowId()
+          + "' has failed on " + azkabanName);
 
       message.println("<h2 style=\"color:#FF0000\"> Execution '" + execId
-          + "' of flow '" + flow.getFlowId() + "' has failed on " + azkabanName
-          + "</h2>");
+          + "' of flow '" + flow.getFlowId() + "' of project '"
+          + flow.getProjectName() + "' has failed on " + azkabanName + "</h2>");
```

Comment:
looks "has encountered a failure on" and "has failed on" are telling the same thing, but having inconsistent expression might cause friction for user to search those emails in their inbox. 
should we have the same string template as a member variable for them?